src/components/MainInfo/MainInfo.tsx

```diff
@@ -0,0 +1,66 @@
+import { useRef, useMemo } from 'react';
+import { Box, Divider, Stack, useMediaQuery } from '@mui/material';
+import { headerHeight, mediaQueries, spacings, tabsHeight } from '../DesignSystem/DesignSystem';
+import MainInfoSidebar from './Sidebar';
+import { getAvailibleData, getMainInfoStructure, scrollMarginTop } from './utils';
+import MainInfoArticle from './Article';
+import useObserver from '../../hooks/useObserver';
+import useAvailableHeight from '../../hooks/useAvailibleHeight';
+import { MainInfoProps } from '../../types/types';
+
+const MainInfo = ({ zaaktype, loading }: MainInfoProps) => {
```

Comment:
Thinking about sven's comments on the other PR: can this be made generic?

Maybe the generation of the `mainInfoStructure` can be done elsewhere and then passed to this component?

Reply:
✅ I will make this more generic.